src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.io.*;
```

Comment:
Perhaps consider importing only the libraries you need instead of *.

Reply:
Thank you for your comments! I have just explicitly listed down my wildcard imports to all my classes.